### DISCUSSION TITLE
Launchscreen.storyboard ios

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#8901] [Q&A]
What are the steps for hooking up the storyboard file in ios? > so far i created one with white Launchescreen and registered it in the info.plist. still i get some dedault one with black launchescreen witch does not exist or i cant find.

**Comment by Daantje007** [comment]
Not realy working.. i took the time to fill all launch photos and icons with photoshop...

**Reply by jeromelaban** [reply]
Have you taken a look at this documentation: https://platform.uno/docs/articles/splash-screen.html
